src/Chronos.hs

```diff
@@ -41,6 +41,7 @@ module Chronos
   , ordinalDateToDay
   , monthDateToDayOfYear
   , dayOfYearToMonthDay
+  , timeOfDayToNanosecondsSinceMidnight
```

Comment:
Needed to expose this to construct the UtcTime; if you want me to user another equivalent type, let me know